Makefile

```diff
@@ -24,8 +24,10 @@ $(BIN_DIR):
 
 
 # build native bbgo
-bbgo:
-	go build -tags web,release -o $(BIN_DIR)/$@ ./cmd/bbgo
+bbgo: static bbgo-full
```

Comment:
can be just:

```
bbgo: static
        go build -tags web,release -o $(BIN_DIR)/bbgo ./cmd/bbgo
```

target bbgo is used to build the default bbgo (for local environment)